WalletWasabi/WabiSabi/Backend/Rounds/Arena.Partial.cs

```diff
@@ -241,7 +252,7 @@ private async Task<ConnectionConfirmationResponse> ConfirmConnectionCoreAsync(Co
 					// Update the coinjoin state, adding the confirmed input.
 					round.CoinjoinState = round.Assert<ConstructionState>().AddInput(alice.Coin, alice.OwnershipProof, round.CoinJoinInputCommitmentData);
 					alice.ConfirmedConnection = true;
-
+					RequestTimeStatista.Instance.Add("connection-confirmation-inside2", DateTimeOffset.UtcNow - beforeInside);
```

Comment:
Reading the logs will give me zero clue what's the difference between `connection-confirmation-inside1` and `connection-confirmation-inside2`. Can we be more precise?

Reply:
These statistics will make sense only if you know the code. I believe aiming for more will open a can of worms plus, the question: do we need that?